.angular-cli.json

```diff
@@ -17,7 +17,7 @@
       "test": "test.ts",
       "tsconfig": "tsconfig.app.json",
       "testTsconfig": "tsconfig.spec.json",
-      "prefix": "app",
+      "prefix": "ngxd",
```

Comment:
why `ngxd` ? the d in `ngxd-duration-picker` looks a repetition :)

Reply:
I agree. Any other suggestions? My first thought was `dp` but decided against it ;) . Could still be a decent prefix though. Could be `sd` your your github name too.

Reply:
what about `ngx` ?

Reply:
for that reason it would be easier to be remembered :D

Reply:
Ok, switched to `ngx`.